docs/ref/models/querysets.txt

```diff
@@ -2470,11 +2470,53 @@ be in conflict must be provided.
 Enabling the ``ignore_conflicts`` parameter disables setting the primary key on
 each model instance (if the database normally supports it).
 
+**Advanced Usage: Conditional Updates with** ``condition``
+
+The ``condition`` parameter, allows specifying a
+conditional SQL WHERE clause for bulk insert operations involving conflict
+resolution. This feature is supported by PostgreSQL and SQLite backends 
+(from SQLite version 3.42.0). ``condition`` accepts Django's ``F``, ``Q``, 
+and ``Excluded`` expressions.
+
+The ``Excluded`` class, extends Django's ``F`` 
+class and is specifically designed to handle conflict resolution in bulk insert 
+operations. Unlike ``F`` expressions, which reference fields in the existing 
+database, ``Excluded`` refers to values in the proposed data for insertion. 
+Moreover, similar to ``F`` expressions, ``Excluded`` supports arithmetic and logical
+operations, making it particularly useful in ``bulk_create(..., update_conflicts=True)``
+scenarios where complex conditions are required.
```

Comment:
I think we should probably document Excluded separately in `docs/ref/models/expressions.txt`
Then we can have people link to the Excluded docs specifically 
This would have a `versionadded` note

Reply:
> I think we should probably document Excluded separately in `docs/ref/models/expressions.txt` Then we can have people link to the Excluded docs specifically This would have a `versionadded` note

who are the people who need to create this link? 
do you have any resources to share with me for this task?

Reply:
The documentation is written in reStructuredText and if you read [Django's guide on writing documentation](https://docs.djangoproject.com/en/5.0/internals/contributing/writing-documentation/) this also tells you how to reference and link things
One of the things I do is check how something similar is documented, find the source code for those docs and use that as a template. 👍